specs/online_order_spec.rb

```diff
@@ -3,88 +3,114 @@
 require 'minitest/skip_dsl'
 
 # TODO: uncomment the next line once you start wave 3
-# require_relative '../lib/online_order'
+require_relative '../lib/online_order'
 # You may also need to require other classes here
 
 # Because an OnlineOrder is a kind of Order, and we've
 # already tested a bunch of functionality on Order,
 # we effectively get all that testing for free! Here we'll
 # only test things that are different.
 
-xdescribe "OnlineOrder" do
+describe "OnlineOrder" do
   describe "#initialize" do
     it "Is a kind of Order" do
-      # Check that an OnlineOrder is in fact a kind of Order
+      all_orders = Grocery::OnlineOrder.all
```

Comment:
Based on the description in the `it` block you should have a test that makes sure that an OnlineOrder is a subclass of Order.

So:
```ruby
sample_order = Grocery::OnlineOrder.new(1, {seeds: 58}, 3, :paid)
sample_order.must_be_instance_of Grocery::Order
```

This should work because an OnlineOrder **is** an Order (inheritance).